inluxclient/query_test.go

```diff
@@ -0,0 +1,1083 @@
+package influxclient_test
+
+import (
+	"errors"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"io/ioutil"
+	"strings"
+	"testing"
+	"time"
+
+	influxclient "github.com/influxdata/influxdb-client-go/inluxclient"
+	"github.com/stretchr/testify/require"
+)
+
+func mustParseTime(s string) time.Time {
```

Comment:
FWIW I think it's nicer to have helper functions at the end of the file so they don't get in the way of top-down reading of the code.

Reply:
Ok